modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -347,6 +346,20 @@ def isCurrentCommitOnMaster() {
   ) == 0
 }
 
+/**
+ * Check whether there is a git branch named release
+ * This test is useful for determining whether we should update this branch or
+ * not
+ */
+def releaseBranchExists() {
+  sshagent(["govuk-ci-ssh-key"]) {
```

Comment:
Does this `sshagent` wrapper return the boolean from lines 356-359? Or does it return some value of its own? I can't find it in the docs.

Reply:
That's a good point. I wasn't too sure and in my testing it worked successfully but I may have only testing the truthy route. I'll give it a spin on something where it is falsy to check.

Reply:
@suzannehamilton Yup I've been able to confirm it returns a boolean: https://ci.integration.publishing.service.gov.uk/job/govuk_app_config/job/master/26/console